README.md

```diff
@@ -118,6 +118,59 @@ logger.read(
     }
 );
 ```
+### readSync(*index*)
+ The *readSync()* method reads objects from the logger synchronously, returning a single log object for the *index* given. *readSync()* works similarly to *read()* - it will return the most recent object when `index == -1` and the oldest object when `index == 1`.  If the absolute value of *index* is greater than the number of logs, *readSync* will return null.
```

Comment:
This description is very confusing. `read()` has `skip` and `step` arguments. `index` doesn't sound like any of them. 
- Why  can't `index` be zero?
- How would you read an element with index 1?